index.js

```diff
@@ -92,21 +93,30 @@ function fastifyWebsocket (fastify, opts, next) {
     done()
   })
 
+  fastify.addHook('onResponse', (request, reply, error, done) => {
+    if (request.ws) {
+      request.raw[kWs].destroy()
+    }
+    done()
+  })
```

Comment:
Worth writing a regression test for this? Is probably quite annoying, I am ok without

Reply:
Without this change, the test would fail. Truth be told, I didn't know why they were passing before.

Reply:
Oh, strange! But ok!